gen-tests.edn

```diff
@@ -0,0 +1,5 @@
+#kaocha/v1
+{:tests [{:id :generated
+          :test-paths  ["target/test-doc-blocks/test"]
+          :kaocha.hooks/post-load-test [lread.test-doc-blocks.kaocha.hooks/fail-on-no-tests-found]}]
+ :randomize? false}
```

Comment:
`:reporter kaocha.report/documentation` might be nice here

Reply:
Let's do that!